### DISCUSSION TITLE
Open vtk in Paraview

### DISCUSSION THREAD

**Discussion by ZdendaH** [OpenRadioss/OpenRadioss#597] [Q&A] [answered]
Dear OpenRadioss,

I would like to ask how to open vtk file in Paraview after I used the anim_to_vtk script. I got the files. Paraview does not open itself and asks to choose file extension. Which file extension to choose?

Thank you.

**Comment by MarianBulla** [comment]
Dear @ZdendaH, 
just open them in ParaView (see screenshot).
Hope it helps you ?
Thanks and best regards,
Marian
![ParaView](https://user-images.githubusercontent.com/105412502/194705142-35c2938b-2958-4b87-9e4b-46f4e45c4f50.png)

**Reply by ZdendaH** [reply]
Thank you for answer. I am new user with Radioss solver and Linux too. I have to learn the Linux command. I used the anim_to_VTK script in bad way.

**Comment by ZdendaH** [comment, marked as answer]
I have another question. After the simulation has finish I got many anim. result files (for example *A000 - AXXX) than the anim_to_VTK script is used but I do it one by one. Is any script(bash, sh) which would run on all anim files to transfer to VTK. Thank you.

**Reply by ZdendaH** [reply]
I found the answer here.

**Reply by MarianBulla** [reply]
Great !
You're becoming an expert!
Thank you for your answer and enjoy solving your models with OpenRadioss :-)
